Podcast/ProfileActivityTableViewCell.swift

```diff
@@ -0,0 +1,120 @@
+                case .Liked:
+                    actText = NSMutableAttributedString(string: "Liked "+podcastName)
+                    actText.addAttributes([NSForegroundColorAttributeName: UIColor.podcastPink, NSFontAttributeName: UIFont.boldSystemFont(ofSize: 14)], range: NSRange(location: 6, length:podcastName.characters.count))
+//                    actText.addAttributes([NSFontAttributeName: UIFont.boldSystemFont(ofSize: 14)], range: NSRange(location: 6+podcastName.characters.count+4, length:seriesName.characters.count))
+                    break
+                case .Listened:
+                    actText = NSMutableAttributedString(string: "Listened to "+podcastName)
+                    actText.addAttributes([NSForegroundColorAttributeName: UIColor.podcastPink, NSFontAttributeName: UIFont.boldSystemFont(ofSize: 14)], range: NSRange(location: 12, length:podcastName.characters.count))
+//                    actText.addAttributes([NSFontAttributeName: UIFont.boldSystemFont(ofSize: 14)], range: NSRange(location: 12+podcastName.characters.count+4, length:seriesName.characters.count))
+                    break
+                }
+                actLabel.attributedText = actText
+            }
+        }
+    }
+    
+    // Mark: Init
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        frame.size.height = height+2*contentPaddingY
+        backgroundColor = UIColor.podcastWhite
+        selectionStyle = .none
+        
+        containerView = UIView(frame: CGRect.zero)
+        containerView.backgroundColor = UIColor.podcastWhiteDark
+        contentView.addSubview(containerView)
+        
+        podImage = UIImageView(frame: CGRect.zero)
+        podImage.image = #imageLiteral(resourceName: "fillerImage")
+        containerView.addSubview(podImage)
+        
+        actLabel = UILabel(frame: CGRect.zero)
+        actLabel.textAlignment = .left
+        actLabel.font = UIFont.systemFont(ofSize: 14)
+        actLabel.textColor = UIColor.black
+        actLabel.numberOfLines = 1
+//        actLabel.backgroundColor = .red
+        containerView.addSubview(actLabel)
+        
+        dateLabel = UILabel(frame: CGRect.zero)
+        dateLabel.textAlignment = .left
+        dateLabel.font = UIFont.systemFont(ofSize: 12)
+        dateLabel.textColor = UIColor.darkGray
+//        dateLabel.backgroundColor = .green
+        containerView.addSubview(dateLabel)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func layoutSubviews() {
+        let cellSize = self.frame.size
+        containerView.frame = CGRect(x: contentPaddingX, y: contentPaddingY, width: cellSize.width-2*contentPaddingX, height: height)
+        
+        let containSize = containerView.frame.size
+        let cellPadding:CGFloat = 12.0
```

Comment:
generally it's good to have a space after `:` when declaring types of variables for better readability